app/src/components/Title.tsx

```diff
@@ -0,0 +1,11 @@
+export default function Title({ text }: { text?: string }) {
+  return (
+    <div className="flex items-center">
+      <div className="bg-blue-950 w-10 h-0.5 mr-2" />
+      <h1 className="uppercase font-mono font-bold text-blue-950">
+        {text ? text : "Default title"}
+      </h1>
+      <div className="bg-blue-950 w-10 h-0.5 ml-2" />
```

Comment:
I think we should have the possibility to change the color and text size from the props

Reply:
Good to know, thank you 👍 however I think the way you wrote it now makes more sense IMO, which allows the consumer to override the className itself, though it is not type-checked. Maybe there is an enum we can use from Tailwind?